### PR TITLE
Fix tests and align version numbers

### PR DESCRIPTION
**Pull request by justinbarry**

### CODE REVIEW

File: packages/augur-ui/jest.config.js [hunk 1]
```diff
@@ -67,11 +67,7 @@ module.exports = {
 
   // A map from regular expressions to module names that allow to stub out resources with a single module
   moduleNameMapper: {
-    "^src/(.*)$": "<rootDir>/src/$1",
-    "^(config|modules|utils|services)/(.*)$": "<rootDir>/src/$1/$2",
-    "^assertions/(.*)$": "<rootDir>/test/assertions/$1",
-    "^test/(.*)$": "<rootDir>/test/$1",
-    "\\.(style|styles)$": "<rootDir>/test/mockStyle.js"
+    "\\.styles?$": "<rootDir>/test/mockStyle.js"
```
Comment (bthaile) [inline packages/augur-ui/jest.config.js]: I think there might be include files that end with `.style` would that be an issue?

File: packages/augur.js/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "augur.js",
-  "version": "6.0.1",
+  "version": "6.0.0-0",
```
Comment (bthaile) [inline packages/augur.js/package.json:3]: I thought this got changed to 6.0.1 ?

File: packages/augur-ui/jest.config.js (hunk 1 above)
Comment (justinbarry) [reply packages/augur-ui/jest.config.js]: That is just shorthand for both style and styles. It was done on another branch and shouldn't haven't show up here.

File: packages/augur.js/package.json (hunk 2 above)
Comment (justinbarry) [reply packages/augur.js/package.json:3]: The version needs to match what is required in the other modules for proper linking.

File: packages/augur-ui/jest.config.js (hunk 1 above)
Comment (justinbarry) [reply packages/augur-ui/jest.config.js]: Will rebase it out.

File: packages/augur.js/package.json (hunk 2 above)
Comment (justinbarry) [reply packages/augur.js/package.json:3]: It breaks this: https://github.com/AugurProject/augur/pull/96#issuecomment-440855225

**Pull request merged**